src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -441,16 +496,33 @@ public Void call() throws Exception {
                             );
                             break;
                         }
+                        if (previousLine != null) {
+                            sb.append(previousLine).append(System.getProperty("line.separator"));
+                            previousLine = null;
```

Comment:
@dean-e-clark again, this PR cannot have `null` at all, it's  billion-dollar mistake :wink: